README.md

```diff
@@ -46,45 +46,27 @@ It’s a Rails app that:
 
 5. Clone this repo
 
-6. If you don’t have the `bundler` Ruby gem, install it:
+6. Run:
 
     ```sh
-    $ gem install bundler
+    $ bin/setup
     ```
 
-7. Wherever you cloned the repo, go to that directory and install dependencies:
+    This will set up and populate a database with sample data and an admin user. (Read the output for details!)
 
-    ```sh
-    $ bundle install --without production
-    ```
-
-8. Copy the .env.example file to .env - this allows for easy configuration locally.
```

Comment:
I feel like we still need a line that at least explains this is what you modify to change your local configuration. That’s an important thing that people need to know how to do.